.github/actions/rebase/action.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Rebase'
+description: 'Rebase against base branch'
+runs:
+  using: "composite"
+  steps:
+    - name: Rebase feature branch with base branch
+      shell: bash
+      run: |
+        git config --global --add safe.directory /__w/candlepin/candlepin
+        git config --global user.email "ghaction@random.com"
+        git config --global user.name "ghaction"
+        
+        base_branch="${{ github.base_ref }}"
+        
+        # Check if this is a pull request and if the base branch has changed
+        if [ -n "$GITHUB_HEAD_REF" ] && [ "$GITHUB_HEAD_REF" != "$base_branch" ]; then
+          git remote add base_branch https://github.com/${GITHUB_REPOSITORY}.git
+          git fetch base_branch
+          git rebase base_branch/$base_branch
```

Comment:
this is a bit confusing to read here. can we leave the $base_branch variable as is, which will contain the value of the target branch (e.g. main), but rename the remote branch alias as `remote_base_branch` or `origin_base_branch`?.

Then this command will read a bit better: `git rebase remote_base_branch/$base_branch`

Reply:
done